control-plane/cmd/kafka-controller/main.go

```diff
@@ -76,22 +76,21 @@ func main() {
 			},
 		},
 
-		// TODO: enable later
-		//// Namespaced broker controller
-		//injection.NamedControllerConstructor{
-		//	Name: "broker-namespaced-controller",
-		//	ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
-		//		return broker.NewNamespacedController(ctx, watcher, brokerEnv)
-		//	},
-		//},
-		//
-		//// Namespaced trigger controller
-		//injection.NamedControllerConstructor{
-		//	Name: "trigger-namespaced-controller",
-		//	ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
-		//		return trigger.NewNamespacedController(ctx, watcher, brokerEnv)
-		//	},
-		//},
+		// Namespaced broker controller
+		injection.NamedControllerConstructor{
+			Name: "broker-namespaced-controller",
+			ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
+				return broker.NewNamespacedController(ctx, watcher, brokerEnv)
+			},
+		},
+
+		// Namespaced trigger controller
+		injection.NamedControllerConstructor{
+			Name: "trigger-namespaced-controller",
+			ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
+				return trigger.NewNamespacedController(ctx, watcher, brokerEnv)
+			},
+		},
```

Comment:
We're enabling these controllers which means that they will be released but I think we should solve the known issues first like:
- https://github.com/knative-sandbox/eventing-kafka-broker/pull/2361#discussion_r927423962
- or the list in https://github.com/knative-sandbox/eventing-kafka-broker/pull/2361

Reply:
/hold

OK we need to make a decision:
- Release with poor CM support
- Block the release until CM issues are completely fixed

Reply:
I'd prefer solving known issues because right now these deployments aren't debuggable at all, no tracing, no logging config or stale ConfigMaps contain buggy behavior